test/Server.spec.js

```diff
@@ -616,6 +616,63 @@ describe('Server', () => {
 		})
 	})
 
+	it('should render comments to string', () => {
+		assert.html(createComment('1st'), '<!--1st-->')
+		assert.html(h('div', 1, createComment('1st'), h('h1', 2)), '<div>1<!--1st--><h1>2</h1></div>')
+	})
+
+	it('should render comments to json', () => {
```

Comment:
I'm not familiar with this usage. Where is this JSON used? (also should children be an array?)

Reply:
The `createComment` public API introduced in 9.0 works much like `createPortal`. That said comments are akin to text nodes so their children can only contain a single textual value.

This tests that comment element types are also handled by the server render. Public usage of this could be with regards to snapshots in testing frameworks.

I think that most of test diffs, `Server.spec.js` included, where just movement diffs. The only new tests i know of are for `createRef`, `forwardRef` and the Error Boundary patch.